wagtailio/project_styleguide/templates/patterns/components/streamfields/highlight_block/highlight_block.html

```diff
@@ -0,0 +1,24 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="highlight{% if value.image_on_right or image_on_right %} highlight--reverse{% endif %} grid">
```

Comment:
Suggestion:

I think we could simplify it to `if image_on_right`, this can be set via the block's context, or when including the template. We should try if possible to keep FE logic as simple and non-bound to a model's variables as possible.

Reply:
Went through this on a call - thanks Will!